recipes/replxx/all/conanfile.py

```diff
@@ -48,6 +48,12 @@ def _configure_cmake(self):
         return self._cmake
 
     def build(self):
+        if tools.Version(self.version) < "0.0.3":
+            tools.replace_in_file(
+                os.path.join(self._source_subfolder, "src/io.cxx"),
+                "#include <array>\n",
+                "#include <array>\n#include <stdexcept>\n"
```

Comment:
It is patched in the same way in the upstream, since 0.0.3 should work fine.

And this is how it looks in current master:
https://github.com/AmokHuginnsson/replxx/blob/master/src/terminal.cxx